test/commands/user/reputation_token/calculate_contextual_data_test.rb

```diff
@@ -17,8 +17,11 @@ class User::ReputationToken::CalculateContextualDataTest < ActiveSupport::TestCa
     5.times { create :user_exercise_contribution_reputation_token, user: }
     6.times { create :user_mentored_reputation_token, user: }
     7.times { create :user_published_solution_reputation_token, user: }
+
+    generate_reputation_periods!
+
     data = User::ReputationToken::CalculateContextualData.(user.id)
-    expected_activity = "1 PR created • 3 PRs reviewed • 2 PRs merged • 4 exercises authored • 5 exercise contributions • 6 solutions mentored • 7 solutions published" # rubocop:disable Layout/LineLength
+    expected_activity = "1 PR created • 5 PRs reviewed and/or merged • 9 exercise contributions • 6 solutions mentored • 7 solutions published" # rubocop:disable Layout/LineLength
```

Comment:
This is the only "lossy" bit of this PR. But it's like 4 OOM faster so I don't care.

Reply:
Yeah that's fine